pkg/cleanup/containers.go

```diff
@@ -134,19 +134,19 @@ func (c *containers) stopContainerd() {
 
 func (c *containers) stopAllContainers() error {
 	var errs []error
-	logrus.Debugf("trying to list all pods")
 
 	var pods []string
 	err := retry.Do(func() error {
+		logrus.Debugf("trying to list all pods")
 		var err error
 		pods, err = c.Config.containerRuntime.ListContainers()
 		if err != nil {
 			return err
 		}
 		return nil
-	})
+	}, retry.Attempts(3))
```

Comment:
The default of 10 times was too much, I guess? If we had a context here, we could support Ctrl+C from the console.